Classes/common/CDTReplicator/CDTReplicator.h

```diff
@@ -16,6 +16,7 @@
 #import <Foundation/Foundation.h>
 
 #import "CDTReplicatorDelegate.h"
+#import "NSURLSessionConfigurationDelegate.h"
```

Comment:
This import needs to be in the form `#import <Foundation/NSURLSessionConfiguration.h>` it's for correctness when we build as a framework